pages/detail/book/[id].js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import Card from "@/components/Card/Card";
+import Bottom from "@/components/Bottom/Bottom";
+
+export default function DetailBookPage({ data }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const currentBook = data.find((book) => book.id == id);
+
+  if (!currentBook) {
+    return <h1>404 An error occured...</h1>;
+  }
+  return (
+    <StyledDetail>
+      <Card book={currentBook} />
+      <StyledCard>
+        <StyledId>
+          <span>ID:</span>
+          <span>{currentBook.id}</span>
+        </StyledId>
+        <h4>Teaser:</h4>
+        <StyledArticle>{currentBook.teaser}</StyledArticle>
+      </StyledCard>
+      <Bottom>
+        <Link href={`../../`} className="link_back">
```

Comment:
your `href` should be relative to the root of your project. If you leave it as it is right now, it is rather confusing to read and it would also be confusing for user working with a screen reader.

this would be better:
```suggestion
        <Link href="/detail">
```